kimchi/Cargo.toml

```diff
@@ -46,15 +45,13 @@ groupmap = { path = "../groupmap", version = "0.1.0" }
 mina-curves = { path = "../curves", version = "0.1.0" }
 o1-utils = { path = "../utils", version = "0.1.0" }
 mina-poseidon = { path = "../poseidon", version = "0.1.0" }
+mina-book = { path = "../book", version = "0.1.0" }
```

Comment:
this line defeats the purpose :o you're still going to rely on the dependencies there

Reply:
Well, our original goal was to make the spec as part of build, so without this it will not build, right?

Reply:
Ah, I see what you mean.. we add it to top level workspace instead.

Reply:
yeah otherwise this still breaks the wasm build